testing/nightly/gpu-operator.sh

```diff
@@ -278,24 +278,86 @@ shift
 
 set -x
 
+prepare_dtk_imagestream() {
```

Comment:
This function appears twice?

Reply:
yes, currently it's part of `run_gpu_operator_with_dtk.sh`, 
but for upgrade testing, I need it again in `gpu_operator.sh` 
and because the former is sourcing the later, previously my modification was overwitten -.-
```
prepare_dtk_imagestream() {

}

source gpu_operator.sh # overwrites ^^^

prepare_dtk_imagestream # not calling the expected function -,-
```

I'll keep only the function to `gpu-operator.sh` to avoid duplication, but that's not the right place to have it, it fits better in `run_gpu-operator_with_dtk.sh`
anyway, that's just WIP as this function won't be necessary when the DTK image will be available on all the OCP versions